resources/case_studies/Insurance/InsuranceClient.obs

```diff
@@ -2,9 +2,27 @@ import "resources/case_studies/Insurance/InsuranceService.obs"
 import "Obsidian Runtime/src/Runtime/edu/cmu/cs/obsidian/java-utilities/System.obs"
 
 main contract InsuranceClient {
-	transaction main(remote InsuranceService i) {
-         Bid b = i.requestBids(3);
-         System.print("bid : 3");
+	transaction main(remote InsuranceService is) {
+	     Insurer ins = new Insurer();
+	     Bank bank = new Bank();
+
+	     // ins. service needs reference to insurer, where it
+	     // then initializes insurer w bank
+	     is.addInsurer(ins);
+	     is.addBankToInsurer(bank);
```

Comment:
This structure is a little weird. I realize in part it's because of limitations in the Obsidian environment for now, but maybe a rename of "addBankToInsurer" would help. That is, an insurer isn't a container for banks.